driver/razercommon.h

```diff
@@ -40,6 +40,7 @@ do { \
 #define VARSTORE         0x01
 
 // LED definitions
+#define MOUSEPAD_LED      0x00
```

Comment:
Maybe call it `ZERO_LED` or something else but this LED is also used for other devices so let's not make it "mousepad"-specific

Reply:
Renamed to ZERO_LED.